content/en/infrastructure/process/_index.md

```diff
@@ -118,7 +118,26 @@ datadog:
 
 [1]: https://github.com/DataDog/helm-charts/blob/master/charts/datadog/values.yaml
 {{% /tab %}}
+{{% tab "ECS Fargate" %}}
 
+In order to collect processes, the Datadog Agent must be running as a container within the Task.
+
+To collect process information in ECS Fargate, add the `PidMode` term to the Task Definition and set it as follows:
+
+```text
+"pidMode": "task"
+```
+
+Once enabled, use the `AWS Fargate` Containers facet on the [Live Processes page][1] to filter processes by ECS or EKS, or enter `fargate:ecs` or `fargate:eks` in the search query.
```

Comment:
Do we need to mention `fargate:eks` here?

Reply:
Actually since this tab is for ECS Fargate - should we drop mentioning EKS entirely?

Reply:
Ah good point, yes we should drop it.